katip/katip.cabal

```diff
@@ -65,7 +65,7 @@ library
                , old-locale >= 1.0
                , string-conv >= 0.1
                , template-haskell >= 2.8
-               , text >= 1.2.4.0 && < 2.0
+               , text >= 1.2.4.0 && < 1.3 || >= 2.0 && < 2.1
```

Comment:
just curious, is there a reason for explicitly setting the upper bound to 1.3?

Reply:
It's a more pedantic choice. At least in theory nothing (except common sense) prevents `text` from releasing 1.3.X.X series, incompatible both with 1.2.X.X and 2.0.X.X.

Reply:
I just tested your branch against text-2.0 and tests are passing. Could you update this bound to be just `text >= 1.2.4.0`? I try to avoid upper bounds as they create a lot of busy work for little benefit until there's actually a known breaking change. When a new dependency comes out that doesn't build anymore, users can pin the latest version that does work and file an issue where we can evaluate if we can support the new version or if we have to put in an upper bound for the time being. I believe I set the `2.0` upper bound because the code was not compatible with `2.0` and at the time it was difficult to get a build plan including `2.0`. I expect with your work here that we should stay compatible with text for some time.

Reply:
@MichaelXavier done.

Reply:
Thanks!